src/main/preload/dropdown.js

```diff
@@ -34,12 +35,15 @@ window.addEventListener('message', async (event) => {
     case CLOSE_TEAMS_DROPDOWN:
         ipcRenderer.send(CLOSE_TEAMS_DROPDOWN);
         break;
+    case UPDATE_TEAMS:
+        ipcRenderer.invoke(UPDATE_TEAMS, event.data.data);
+        break;
     default:
         console.log(`got a message: ${event}`);
         console.log(event);
     }
 });
 
-ipcRenderer.on(UPDATE_TEAMS_DROPDOWN, (event, teams, activeTeam, darkMode, expired, mentions, unreads) => {
```

Comment:
075 but would it make sense to add a GPO field to the server info? that way you could potentially reorder non-gpo servers while disallowing drag and drop on the others.

This might be making everything more complex so it could be something to do in a future PR

Reply:
Yeah there's a bit more complexity to that, I think we need to evaluate how we deal with GPO teams in the first place. I'd like to actually maybe add them to the user stored config (or at least their ordering) so that they can be re-ordered. But that's another problem to solve another time.